client/services/code_evaluators/CodeRunnerDispatcherServiceSpec.js

```
@@ -0,0 +1,36 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for CodeRunnerDispatcherService
+ */
+
+describe('CodeRunnerDispatcherService', function() {
+  var CodeRunnerDispatcherService;
+
+  beforeEach(module('tie'));
+  beforeEach(inject(function($injector) {
+    CodeRunnerDispatcherService = $injector.get('CodeRunnerDispatcherService');
+  }));
+
+  describe('pythonCodeRunnerService', function() {
+    it('should return error if not using Python language', function() {
```

should throw an error if passed a non-Python language